llvm/test/Feature/Repo/repo_hash_graph_dependencies.ll

```diff
@@ -0,0 +1,27 @@
+; Test GO's information of Dependencies - global variable.
+; A global variable `FPtr` has a reference to a function `_Z3addii`, `_Z3addii` will be put into the Dependencies of `FPtr`.
```

Comment:
Could we avoid using mangled names here(they just make it harder to read): call the function “add”?

Reply:
Changed in commit cca1f2d.